lib/tasks/backfill_qualification_public_ids.rake

```diff
@@ -0,0 +1,6 @@
+desc 'Sets the public_id for qualifications that do not yet have one'
+task backfill_qualification_public_ids: :environment do
+  ApplicationQualification.where(public_id: nil).each do |qualification|
```

Comment:
We will have to run this multiple times, to ensure everything is backfilled. Second run should be very short